README.md

```diff
@@ -85,6 +85,7 @@ Awesome Wagtail [![Awesome](https://cdn.rawgit.com/sindresorhus/awesome/d7305f38
 - [wagtail-generic-chooser](https://github.com/wagtail/wagtail-generic-chooser) - provides base classes for building chooser popups and form widgets for the Wagtail admin, matching the look and feel of Wagtail's built-in choosers for pages, documents, snippets and images.
 - [wagtail-multi-upload](https://github.com/spapas/wagtail-multi-upload) - allows uploading of multiple related images for a page.
 - [wagtail-color-panel](https://github.com/marteinn/wagtail-color-panel) - Introduces panels for selecting colors in Wagtail.
+- [Wagtail Ace Editor](https://github.com/Nigel2392/wagtail_ace_editor) - An IDE- like code editor right in your Wagtail admin!
```

Comment:
```suggestion
- [Wagtail Ace Editor](https://github.com/Nigel2392/wagtail_ace_editor) - An IDE-like code editor right in your Wagtail admin.
```

Minor typo and let's use a simple sentence.

Thanks though, I will patch this in and merge.

Be sure to add the Package to the Wagtail classifier on PIP if you have not already done this https://pypi.org/search/?c=Framework+%3A%3A+Wagtail+%3A%3A+6